podium/apollo-map-props
# Upgrade Apollo Client to V3

Currently upgrading my-precious to use [apollo client 3](https://www.npmjs.com/package/@apollo/client). When adding @apollo/client as the provider, apollo-client consumer does not know about the client because it is different. 

How to test:

1. run `npm run build` in this repo
2. copy and paste this repo's 
```
build/withClient.js to client repo's node_modules/apollo-map-props/build/withClient.js
build/withMutation.js to client repo's node_modules/apollo-map-props/build/withMutation.js
build/withQuery.js to client repo's node_modules/apollo-map-props/build/withQuery.js
```

3. Go to my-precious [branch](https://gitlab.podium.com/engineering/admin/my-precious/-/merge_requests/1086). Everything should just work fine.
